python/feathub/processors/tests/mixed_transform_test_base.py

```diff
@@ -0,0 +1,781 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from abc import abstractmethod
+from datetime import timedelta
+from math import sqrt
+
+import pandas as pd
+
+from feathub.common.test_utils import to_epoch_millis
+from feathub.common.types import Float64, String, Int64
+from feathub.feathub_client import FeathubClient
+from feathub.feature_views.derived_feature_view import DerivedFeatureView
+from feathub.feature_views.feature import Feature
+from feathub.feature_views.sliding_feature_view import SlidingFeatureView
+from feathub.feature_views.transforms.over_window_transform import OverWindowTransform
+from feathub.feature_views.transforms.python_udf_transform import PythonUdfTransform
+from feathub.feature_views.transforms.sliding_window_transform import (
+    SlidingWindowTransform,
+)
+from feathub.processors.tests.feathub_test_base import FeathubTestBase
+from feathub.processors.tests.sliding_window_transform_test_base import (
+    ENABLE_EMPTY_WINDOW_OUTPUT_SKIP_SAME_WINDOW_OUTPUT,
+    DISABLE_EMPTY_WINDOW_OUTPUT_WITHOUT_SKIP_SAME_WINDOW_OUTPUT,
+    ENABLE_EMPTY_WINDOW_OUTPUT_WITHOUT_SKIP_SAME_WINDOW_OUTPUT,
+)
+from feathub.table.schema import Schema
+
+
+class MixedTransformTestBase(FeathubTestBase):
```

Comment:
It is probably better to move tests in this class to class with more informative names (e.g. OverWindowTransformTestBase). It is OK for a test in `OverWindowTransformTestBase` to also use other Transforms. And some information is better than no information.